python/hsfs/core/external_feature_group_engine.py

```diff
@@ -80,7 +80,12 @@ def insert(
         )
 
         if ge_report is not None and ge_report.ingestion_result == "REJECTED":
-            return None, ge_report
+            raise DataValidationException(
+                "Data validation failed while validation ingestion policy set to strict, "
+                + f"insertion to {feature_group.name} was aborted.\n"
+                + f"Check a summary of your report at {self._get_feature_group_url(feature_group)}, "
+                + f"or download the full report from {ge_report._full_report_path}."
```

Comment:
Seems to me that the `ge_report._full_report_path` returns the path on the file system - so this error seems a bit vague as the user won't be able to click on it to download the report. 
Maybe we should redirect the user to the feature group overview UI where they can see the result?

Reply:
Good catch, that does sound more helpful